File: 프로그래머스/lv0/120884. 치킨 쿠폰/치킨 쿠폰.py
def solution(chicken):
    #총 쿠폰 수 계산(chicken: 서비스로 받게되는 쿠폰 수, rem: 남는 쿠폰, answer: 총 서비스 수)
    answer = 0
    div = 0
    rem = 0
    while chicken//10!=0:
        div = chicken//10
        rem = chicken%10
        answer+=div
        chicken = div+rem

    #최대 서비스 개수 반환 
    return answer

File: 프로그래머스/lv0/120884. 치킨 쿠폰/치킨 쿠폰.js
function solution(chicken) {
    var answer = 0, cou = chicken;
    //쿠폰 수가 10개 이하가 될 때까지 계속 10을 빼주고, answer 1 더하고, 쿠폰 수 1더해
    while (cou>=10){
        cou-=10;
        answer+=1;
        cou+=1
    }
    return answer;
}